app/admin_statistics.py

```diff
@@ -17,38 +17,30 @@
 import sys
 
 import model
-from utils import *
+import utils
 
-class Handler(BaseHandler):
+
+class Handler(utils.BaseHandler):
+    # Show numbers of new records and notes created in each repository.
```

Comment:
Use """...""" for class document comment.

Also, we are going to add more things in this page, so let's write more general description e.g., "show various statistics for each repository". Also, please mention that it can show statistics for repositories which are no longer active, to explain the difference from the dashboard.